### PR TITLE
ENH - Nebari landing page copy

### PR DESCRIPTION
**Pull request by trallard**
This PR adds the copy (_only_) for the new nebari landing page ([currently in this Figma file](https://www.figma.com/file/48vtmwvvuw8OjRV6Gp9bk9/Nebari-website?node-id=0%3A1))

The actual landing page will be worked on once the text is agreed on

🏗️  WIP - minor wording and approval needed atm 🏗️ 

## Reference Issues or PRs
https://github.com/nebari-dev/nebari-docs/issues/182

### CODE REVIEW

File: docs/src/pages/index.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Tagline: An opinionated open sourc eplatform for Data Science and research teams. Built for scale, designed for collaboration.
+
+## Why choose Nebari?
+
+Nebari provides a unified platform for all users to collaborate on data science and research projects.
+It is designed to be flexible and extensible, and built entirely on open source infrastructure.
+Nebari is vendor-and-cloud-agnostic, and is designed and developed by and for the data science community.
+
+<!-- Need to highlight 6 features -->
+<!-- Added here left-to-right, top-to-bottom -->
```
Comment (trallard) [inline docs/src/pages/index.mdx:18]: Should be in the format of:

1. Feature (couple of words max)
   One-liner about said feature 
   
  See [Dask site](https://www.dask.org/) for an example

File: docs/src/pages/index.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Nebari Homepage
+---
+
+<!-- Copy for Nebari Landing page. See Firma file: https://www.figma.com/file/ccOw2sxgjpEUjrOgKqVXRq/Nebari-website?node-id=809%3A3439 -->
+
+## Hero
+
+Tagline: An opinionated open source platform for Data Science and research teams. Built for scale, designed for collaboration.
```
[the remark this answers is not in the document]
Comment (pavithraes) [reply docs/src/pages/index.mdx:9]: I LOVE the second sentence, I think it condenses the Nebari value prop beautifully.

I'd prefer to keep the tagline minimal. I think both "opinionated" and "open source" can be a part of the next section "Why choose Nebari?" -- what do your think?

How about:

> A data science platform: Built for scale, designed for collaboration.

[the remark this answers is not in the document]
Comment (trallard) [reply docs/src/pages/index.mdx:9]: 🤔  the latter sounds a bit impersonal... what about 

> Your open source Data Science platform: Built for scale, designed for collaboration.

(I know we mention open source later - *but* it never hurts having this front, left, and centre)

File: docs/src/pages/index.mdx (hunk 1 above)
Comment (trallard) [reply docs/src/pages/index.mdx:18]: Suggestion "Power of Dask" -> "Dask powered"

File: docs/src/pages/index.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+1. Extensible/customisable
+   We leverage open source infrastructure and tools in our ecosystem to ensure that you can customise your deployment to your needs.
+1. Reproducible environments
+
+## Deploy anywhere
+
+## Integrations
+
+Nebari supports multiple integrations with tools
+in the open source Data Science ecosystem out of the box.
```
Comment (trallard) [inline docs/src/pages/index.mdx:37]:
```suggestion
Nebari comes with out-of-the-box integrations to multiple tools in the 
Data Science ecosystem.
```

File: docs/src/pages/index.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pavithraes) [reply docs/src/pages/index.mdx:9]: I like it! :)

File: docs/src/pages/index.mdx (hunk 1 above)
Comment (pavithraes) [reply docs/src/pages/index.mdx:18]: I'm done updating the copy!

> Suggestion "Power of Dask" -> "Dask powered"
@trallard Oh, nice! I went with "Powerd by Dask", but I also like "Dask powered" -- feel free to select the one you prefer!

File: docs/src/pages/index.mdx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+<!-- Copy for Nebari Landing page. See Firma file: https://www.figma.com/file/ccOw2sxgjpEUjrOgKqVXRq/Nebari-website?node-id=809%3A3439 -->
+
+## Hero
+
+Tagline: An opinionated open source platform for Data Science and research teams. Built for scale, designed for collaboration.
+
+## Why choose Nebari?
+
+Nebari provides a unified platform for all users to collaborate on data science and research projects.
```
[the remark this answers is not in the document]
Comment (pavithraes) [reply docs/src/pages/index.mdx:13]: I think we can keep it all lowercase? I say this mainly based on [Wikipedia](https://en.wikipedia.org/wiki/Data_science) and [this English stack exchange comment with references](https://english.stackexchange.com/a/223162).

It was interesting that the Google and Microsoft style guides didn't include a capitalization reference for data science (neither did [the few other style guides I looked at](https://www.writethedocs.org/guide/writing/style-guides/)).

File: docs/src/pages/index.mdx (hunk 2 above)
Comment (trallard) [inline docs/src/pages/index.mdx:9]:
```suggestion
Tagline: Your open source data science platform. Built for scale, designed for collaboration.
```
Changed!

File: docs/src/pages/index.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (trallard) [reply docs/src/pages/index.mdx:13]: ok let's not capitalise then

File: docs/src/pages/index.mdx (hunk 1 above)
Comment (trallard) [reply docs/src/pages/index.mdx:18]: I like Dask powered best!

Comment (trallard) [reply docs/src/pages/index.mdx:18]: Integrated @pavithraes wording + some minor edits

File: docs/src/pages/index.mdx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+   Nebari ships with Dask so you can scale your work to terabytes of data, leverage cloud instances with GPUs, and take advantage of adaptive scaling for managing costs.
+
+1. Fully customizable
+   Built with open source infrastructure and tools to give you complete flexibility over your deployment and fit your team's specific needs.
+
+## Deploy anywhere
+
+Try Nebari on your local machine or deploy it on your cloud of choice. Nebari is designed to be flexible, extensible, and vendor-agnostic.
+Nebari can be seamlessly deployed to the major public cloud providers, including AWS, Azure, GCP, and Digital Ocean or to bare-metal HPC clusters.
+
```
[the remark this answers is not in the document]
Comment (pavithraes) [reply docs/src/pages/index.mdx:42]: It sounds good to me!

**Comment by trallard** [comment]
Since the copy seems to be finalised, I am closing this PR.
I have merged the content in #184 where @gabalafou  and I will be working on the landing page

**Pull request closed without merging**